src/math.rs

```diff
@@ -121,3 +121,128 @@ impl MaybeMath<Option<f32>, f32> for f32 {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    mod lhs_option_f32_rhs_option_f32 {
+
+        use crate::math::MaybeMath;
+        use rstest::rstest;
+
+        #[rstest]
+        #[case(Some(3.0), Some(5.0), Some(3.0))]
+        #[case(Some(5.0), Some(3.0), Some(3.0))]
+        #[case(Some(3.0), None, Some(3.0))]
+        #[case(None, Some(3.0), None)]
+        #[case(None, None, None)]
+        fn test_maybe_min(#[case] lhs: Option<f32>, #[case] rhs: Option<f32>, #[case] expected: Option<f32>) {
+            assert_eq!(lhs.maybe_min(rhs), expected);
+        }
+
+        #[rstest]
+        #[case(Some(3.0), Some(5.0), Some(5.0))]
+        #[case(Some(5.0), Some(3.0), Some(5.0))]
+        #[case(Some(3.0), None, Some(3.0))]
+        #[case(None, Some(3.0), None)]
```

Comment:
```suggestion
        #[case(None, Some(3.0), None)] // Note here
```
I don't think we need doc explaining what everything is doing, but this particular case is either incorrect or requires special attention. My intuition says that the "max between None and 3" should be 3. But I'm pretty changing the behaviour would break layout

Reply:
This is already documented in the trait description:
<https://github.com/DioxusLabs/taffy/blob/0061666f68cd5c0ef3144c79c5389c797953a3ab/src/math.rs#L3-L6>

Reply:
Good point, in that case the note maybe isn't as needed.

Reply:
Yep, I think this particular trait follows JS null-handling logic?

I really don't love that it's not commutative, but that's what the existing code base needs.